test_tortoise_interpreter.py

```diff
@@ -62,17 +62,19 @@ def test_sanitized_programs(program, expected):
 
 def test_navigate_command_functions():
     def underscore(command, pen_state):
-        return '___', pen_state
+        pen_state = "UP"
```

Comment:
This doesn't use the `pen_state` argument. It uses the _name_ "pen_state", but it never reads the argument it's given. 

The reason I want the test handlers to use the `command` and `pen_state` arguments is to prove that `navigate_command_functions` is passing in the correct information.